### PR TITLE
Switch to mox3 mocking library

### PR DESCRIPTION
**Pull request by JocelynDelalande**
As for python-slimta

### CODE REVIEW

**Comment by icgood** [comment]
ok to test

**Comment by slimtaci** [comment]
Can one of the admins verify this patch?

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
       install_requires=['python-slimta',
                         'celery'],
       tests_require=['nose',
-                     'mox'],
+                     'mox3'],
```
Comment (icgood) [inline setup.py]: Going forward I'd like to remove the `tests_require` bit and replace it with a `test/requirements.txt` file. Mind doing that? I just made sure Jenkins will pick it up.

**Comment by icgood** [comment]
retest this please

**Pull request merged**